WordPress/Classes/ViewRelated/System/WPTabBarController.m

```diff
@@ -545,11 +516,6 @@ - (void)showPostTabForBlog:(Blog *)blog
         [self showPostTabAnimated:YES toMedia:NO blog:blog];
     }
 }
-// will be removed when the new IA implementation completes
-- (void)showMeTab
```

Comment:
this should probably be removed in the header too, and also line 915 in `WPTabBarController.m`:

`[UIKeyCommand keyCommandWithInput:@"3" modifierFlags:UIKeyModifierCommand action:@selector(showMeTab) discoverabilityTitle:NSLocalizedString(@"Me", @"The accessibility value of the me tab.")]`

Reply:
I think we may want to add those key commands back at some point but with different keys.